lib/core/Actions.js

```diff
@@ -161,23 +165,29 @@ class Actions {
   async search(args = {}) {
     const data = {};
     
-    args.query &&
-      (data.query = args.query);
+	if (args.query) {
+		data.query = args.query;
+	}
     
-    args.ctoken &&
-      (data.continuation = args.ctoken);
+	if (args.ctoken) {
+		data.continuation = args.ctoken;
+	}
     
-    args.params &&
-      (data.params = args.params);
+    if (args.params) {
+		data.params
+	}
     
     if (args.filters) {
-      args.client == 'YTMUSIC' &&
-        (data.params = Proto.encodeMusicSearchFilters(args.filters)) ||
-        (data.params = Proto.encodeSearchFilters(args.filters));
+		if (args.client == 'YTMUSIC') {
+			data.filters = Proto.encodeYTMusicFilters(args.filters);
+		} else {
+			data.filters = Proto.encodeFilters(args.filters);
```

Comment:
Seems like the protobuf functions here are incorrect, which is why some tests are failing.

Reply:
Yeah, I blame github copilot for this one :P

Reply:
Yeah, I blame github copilot for this one :P